libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -532,12 +532,14 @@ void NavEKF2_core::readGpsData()
 
 // read the delta angle and corresponding time interval from the IMU
 // return false if data is not available
-bool NavEKF2_core::readDeltaAngle(uint8_t ins_index, Vector3f &dAng) {
+bool NavEKF2_core::readDeltaAngle(uint8_t ins_index, Vector3f &dAng, float &dAng_dt) {
     const AP_InertialSensor &ins = _ahrs->get_ins();
 
     if (ins_index < ins.get_gyro_count()) {
         ins.get_delta_angle(ins_index,dAng);
         frontend->logging.log_imu = true;
+        dAng_dt = MAX(ins.get_delta_angle_dt(imu_index),1.0e-4f);
+        dAng_dt = MIN(dt,1.0e-1f);
```

Comment:
Is `dt` used on purpose here? I don't know if `dt < 1.0e-4f` can ever happen; if it can, then the MAX constraint above is lost.